pkgs/top-level/python-packages.nix

```diff
@@ -9114,6 +9114,27 @@ in {
 
   };
 
+  pytun = buildPythonPackage rec {
+    name = "pytun-${version}";
+    version = "2.2.1";
+    rev = "v${version}";
+
```

Comment:
Please add `doCheck = false;` if there are no tests, which can be run without root privileges.